_harp/about/_company.md

```diff
@@ -1,11 +1,19 @@
 # About us
 
-Founders & Coders C.I.C. runs the only free full-time coding course in the UK. We have developed a highly effective peer-led model of learning which, after only two years of operation, attracts exceptional women and men from all over the world. 30/31 of our most recent graduates are working as software developers.
+Founders & Coders runs a well-established coding camp in London. We are delighted to announce that in 2017 we will be opening our first international campus in the Arab Israeli city of Nazareth.
 
-After completing the course, students are encouraged to join our freelance community and to contribute to a growing array of community activities, including the course itself.
+We follow a highly-effective peer-led approach to learning and we do not employ any teaching or administrative staff. Our low cost of operation allows us to offer our training without charging any fees to our students.
```

Comment:
I think this is a little misleading. What allows us to offer the training for free is our drive to do just that and the blood sweat and tears we put into it - it's actually pretty hard to offer what we do for free! It's really about us striving to offer this for free because of our mission (right up here would be a good place to define it) rather than it being a default position. This seems a little too simplistic. (I'm sorry, I'm on a deadline so can't sit and think of a better solution just yet though 😞 )